packages/autocomplete-core/src/resolve.ts

```diff
@@ -90,15 +91,15 @@ export function resolve<TItem extends BaseItem>(
       return acc;
     }
 
-    const { searchClient, execute, requests } = current;
+    const { searchClient, execute, requesterId, requests } = current;
 
     const container = acc.find<PackedDescription<TItem>>(
       (item): item is PackedDescription<TItem> => {
         return (
           isDescription(current) &&
           isDescription(item) &&
           item.searchClient === searchClient &&
-          item.execute === execute
+          item.requesterId === requesterId
```

Comment:
For now, new requesters that didn't pass a `requesterId` would have it `undefined`, meaning this condition could be truthy for different requesters using the same client and just didn't pass the parameter. Should we also check `requesterId` for truthiness?

Reply:
If we check for truthiness, won't that be the same? `item.requesterId && requesterId ?  item.requesterId === requesterId : true` is equivalent to ` item.requesterId === requesterId` in those cases right?

Reply:
Added a conditional that only batches if requesterId is defined in bbaf5f89